### PR TITLE
Jean S Ariza

### PR DESCRIPTION
**Pull request by jesuarva**
### Set up
   - [x] Use yesterday’s project as the base of this project
   - [x] You _can_ update the HTML/CSS/JS 

### Task
   - [x] Create 10 _unique_ event listeners
   - [x]  Use these events to update the DOM in creative ways

### Stretch 
   - [ ] Stretch Task - go look at GSAP
   - [ ] Stretch assignment

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  <script>
+    /** DOMContentLoaded : alert the time the DOM takes to get loaded */
+    // 3. 'DOMContentLoaded'
+
+    const start = Date.now();
+    document.addEventListener("DOMContentLoaded", function(event) {
+        const elapse = Date.now() - start;
+        console.log(`${event.type} in ${elapse/1000} sg`);
+        alert(`${event.type} in ${elapse/1000} sg`);
+    });
```
Comment (nphillips78) [inline index.html]: Good! Current date AND time elapsed. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    <p></p>
+                </div>
+            </div>
+
+        </section>
+        
+        <!-- <video> -->
+        <video controls id="myvideo">
+                <source src="video/zombie.mp4"></source>
+        </video>
```
Comment (nphillips78) [inline index.html]: Nice

File: index.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+        <footer>
+            <p></p>
+        </footer>
+
+    </div><!-- container -->
+    <div class="container off-container offline">
+        <h2>You just run out off internet conection</h2>
+        <h4>Do not worry, we'll beback as soon as your conection is back.</h4>
+        <img src="img/senal-fuera-de-servicio.jpg" alt="">
```
Comment (nphillips78) [inline index.html]: Excellent!

File: js/index.js [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+ * toggle 'display: none' on random <div.text-content> Elements
+ */
+/** I will add this to my DOM-II events */
+const allContent = topTextContent.concat(bottomTextContent);
+// console.log(allContent);
+function toggleMainContent () {
+  const randomNode = Math.floor(Math.random()*allContent.length);
+  const item = allContent[randomNode];
+  item.classList.toggle('no-visible') ? item.style.display = 'none' : item.style.display = 'block';
+}
```
Comment (nphillips78) [inline js/index.js]: Great use of toggle

File: js/index.js [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+// 1. 'ofline' event.
+bodyContainer,addEventListener('offline', () => {
+  bodyContainer.classList.toggle('offline');
+  document.querySelector('.off-container').classList.toggle('offline');
+})
+// 2. 'online' event.
+bodyContainer,addEventListener('online', () => {
+  bodyContainer.classList.remove('offline');
+  document.querySelector('.off-container').classList.toggle('offline');
+})
```
Comment (nphillips78) [inline js/index.js]: More toggle 👍

File: js/index.js [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+  mainMiddle.style.fontSize = "0.1%";
+  mainBottom.style.fontSize = "0.1%";
+})
+// afterprint
+window.addEventListener('afterprint', (e) => {
+  // console.log(e);
+  body.style.fontSize = bodyFontSize;
+  mainTop.style.fontSize = mainTopFontSize;
+  mainMiddle.style.fontSize = mainMiddleFontSize;
+  mainBottom.style.fontSize = mainBottomFontSize;
```
Comment (nphillips78) [inline js/index.js]: Nice

File: js/index.js [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+/** 'play' && 'pause': change <body> background */
+let video = document.querySelector('video');
+video.addEventListener('play', e => {
+  console.log(e.type);
+  body.style.backgroundColor = "black";
+})
+video.addEventListener('pause', e => {
+  console.log(e.type);
+  body.style.backgroundColor = "white";
+})
```
Comment (nphillips78) [inline js/index.js]: VERY good.

File: index.html (hunk 2 above)
Comment (jesuarva) [reply index.html]: Did you see the video?

**Comment by jesuarva** [comment]
Thank you Cole for your comments!

File: index.html (hunk 2 above)
Comment (nphillips78) [reply index.html]: Yes! It worked 😄

**Pull request closed without merging**